src/riak_exporter/exporter.py

```diff
@@ -89,9 +89,25 @@ def parse_riak_repl_stats_data(self, riak_repl_stats_data):
                     for ckey, cvalue in value[cluster].items():
                         if isinstance(cvalue, (int, float, bool)):
                             prom_value = float(cvalue)
-                            prom_str = "riak_repl_{}{{cluster=\"{}\"}} {}".format(ckey, cluster, prom_value)
+                            prom_str = "riak_repl_fullsync_coordinator_{}{{cluster=\"{}\"}} {}".format(ckey, cluster, prom_value)
                             yield prom_str
 
+            if key == "realtime_queue_stats":
+                for rkey, rvalue in value.items():
+                    if isinstance(rvalue, (int, float, bool)):
+                        prom_value = float(rvalue)
+                        prom_str = "riak_repl_realtime_queue_stats_{} {}".format(rkey, prom_value)
+                        yield prom_str
+
+                    if rkey == "consumers":
+                        for cluster in rvalue:
+                            for ckey, cvalue in value[cluster].items():
+                                if isinstance(cvalue, (int, float, bool)):
+                                    prom_value = float(cvalue)
+                                    prom_str = "riak_repl_realtime_queue_stats_consumers_{}{{cluster=\"{}\"}} {}".format(rkey, cluster, prom_value)
+                                    yield prom_str
```

Comment:
Not asking for it now, but this sequence seems to be repeated many times, maybe we can make a function of it.

Reply:
It's not that easy due to the string and the number of arguments, so I didn't spend time on it